app/questions.py

```diff
@@ -1,71 +1,87 @@
+"""this module has routes for questions and answers"""
+
 from flask_restplus import Resource
 from flask_restplus import reqparse
 from app.models import Models
 from utils.validator import Validate as validate
 
 
 class Questions(Resource):
+    """class for question routes"""
+
     def __init__(self, url):
+        """init method for Question class"""
         self.model = Models()
         self.postquestion_arg = reqparse.RequestParser()
         self.postquestion_arg.add_argument('title',
-                                 type=str, required=True,
-                                 help='title is required!'
-                                )
+                                           type=str, required=True,
+                                           help='title is required!')
         self.postquestion_arg.add_argument('body',
-                                 type=str, required=True,
-                                 help="body is required"
-                                )
+                                           type=str, required=True,
+                                           help="body is required")
 
     def post(self):
+        """post a question"""
```

Comment:
Same to this one, it'd be great if you specified the structure of the question to be posted.